include/modules/wlr/taskbar.hpp

```diff
@@ -40,6 +40,8 @@ class Task {
     FULLSCREEN = (1 << 3),
     INVALID = (1 << 4)
   };
+  // made public so TaskBar can reorder based on configuration.
+  Gtk::Button button_;
```

Comment:
button_ really need to be public?
If yes please remove the _ :)

Reply:
Button on this Task class needs to be public or else TaskBar class cannot access it to sort and move the button. 

There is an alternative way which I mention in the PR comments, but I thought this was cleaner. 

Ill remove the underscore as well, missed that.

Reply:
updated.